extensions/jing-kou-ling/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "jing-kou-ling",
+  "title": "JD code",
+  "description": "generate and parse JD code",
+  "icon": "jdCodeIcon.png",
+  "author": "zooPanda",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "enjcode",
+      "title": "generate JD code",
+      "subtitle": "generate JD code",
```

Comment:
Use subtitles to add context to your command. Usually, it's an app or service name that you integrate with. It makes command names more lightweight and removes the need to specify a service name in the command title.